.github/workflows/prow-action.yml

```diff
@@ -11,10 +11,19 @@ jobs:
     steps:
       - uses: jpmcb/prow-github-actions@v1.1.3
         with:
-          # Only support /kind command for now.
           # TODO: before allowing the /lgtm command, see if we can block merging if changelog labels are missing.
-          prow-commands: "/area
-            /kind
+          prow-commands: |
+            /approve
```

Comment:
FYI approve would require an OWNERS file.
https://github.com/vmware-tanzu/velero/issues/7783

Reply:
I suggest keeping the existing commands. I will create a new PR to introduce other PROW commands.
```
/area
/kind
/cc
/uncc
```